content/pages/health-care/apply.md

```diff
@@ -7,7 +7,7 @@ plainlanguage: 11-02-16 certified in compliance with the Plain Writing Act
 lastupdate: 2017-08-28
 collection: healthCare
 order: 3
-description: Applying for VA health care benefits is easy. Find out which documents you’ll need, and start your online application today.
+description: Find out what documents you'll need to apply for VA health care benefits. Apply online for VA health care benefits today.
```

Comment:
@bethpotts Removed a second `description` block here, and confirmed the correct one was added. Verified the page builds correctly on localhost, and all build tests are now passing.